.github/workflows/ci.yaml

```
@@ -1,108 +1,219 @@
+          name: product-catalog-binary
+          path: ${{ env.SERVICE_PATH }}
+
+      - name: Get Build Version
+        id: vars
+        run: |
+          VERSION=build-${GITHUB_RUN_NUMBER}
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Docker Login
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)